components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -407,6 +449,19 @@ Delete individual existing nodes by selecting the ellipses icon {{< fontawesome
 
 {{< figure src="/images/automate/delete-a-node.png" alt="Delete a Node">}}
 
+### Policyfiles
+
+A [Policyfile]({{< relref "/policyfile" >}}) is an optional way to manage role, environment, and community cookbook data with a single document that is uploaded to the Chef Infra Server. Policyfiles make things easier to test and promote code safely with a simpler interface.
```

Comment:
I thought this was in a comment, but I don't see it here. Policyfiles are the **preferred** way of managing roles, environments, etc. 

role & environment should be plural.